tests/app/views/contexts/test_list_collector.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from app.views.contexts import list_collector
+
+
+def test_build_list_collector_context(
+    rendered_block, answer_store, list_store, form, app
+):
+    context = list_collector.build_list_collector_context(
+        rendered_block, answer_store, list_store, 'en', form
+    )
+
+    assert {'list_items', 'add_link'} <= {*context.keys()}
```

Comment:
As we spoke about, this is doing a set comparison where the keys() can be a superset. Probably need to be checking they're the same?

Reply:
The comparison operator here checks that the left hand set is a subset of the righthand: https://docs.python.org/3.7/library/stdtypes.html#frozenset.issubset

I'm happy with that :+1:

Reply:
I think that a clearer approach would be to assert the full structure returned as context is as expected, as we're doing with test_hub_context.py.